Soccar [Library]/src/nl/soccar/library/Game.java

```diff
@@ -215,10 +215,9 @@ private int getDurationInSeconds() {
                 return 180;
             case MINUTES_5:
                 return 300;
+            default:

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this default to the end of the switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)